modules/network-observability-auth-multi-tenancy.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-auth-mutli-tenancy_{context}"]
+= Configure authorization for multi-tenancy
+Define `ClusterRole` and `ClusterRoleBinding`. The `netobserv-reader` in the `ClusterRole` can be used to enable multi-tenancy and allows individual user access, or group access, to the flows stored in Loki. You can create a YAML file to define these roles.  
+
+.Procedure
+
+. Using the web console, click the Import icon, *+*. 
+. Drop your YAML file into the editor and click *Create*: 
++
+[source, yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: netobserv-reader    <1>
+rules:
+- apiGroups:
+  - 'loki.grafana.com'
+  resources:
+  - network
+  resourceNames:
+  - logs
+  verbs:
+  - 'get'
+...
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: netobserv-writer
+rules:
+- apiGroups:
+  - 'loki.grafana.com'
+  resources:
+  - network
+  resourceNames:
+  - logs
+  verbs:
+  - 'create'
+...
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: netobserv-writer-flp
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: netobserv-writer
+subjects:
+- kind: ServiceAccount
+  name: flowlogs-pipeline    <2>
+  namespace: netobserv
+- kind: ServiceAccount
+  name: flowlogs-pipeline-transformer
+  namespace: netobserv
+----
+<1> This role can be used for multi-tenancy. 
+<2> The `flowlogs-pipeline` writes to Loki. If you are using Kafka, this value is `flowlogs-pipeline-transformer`.
+. Authorize reading permission to `user1` by running the following command: 
```

Comment:
Following my previous comment, I think only the first two items should be kept from this listing: 1.  Click on "+" 2. Add the YAML
And we would move the 3. to an inner section dedicated to Multi-tenancy, that comprises what you wrote in `modules/network-observability-multitenancy.adoc` ?